File: app/src/main/java/com/example/nejat/weatherapp/POJO/Weather.java
package com.example.nejat.weatherapp.POJO;

import com.google.gson.annotations.SerializedName;

public class Weather {
    @SerializedName("main")
    private String weatherFlow;
    private String description;
    @SerializedName("id")
    private int icon;

    public Weather(String weatherFlow, String description, int icon) {

        this.weatherFlow = weatherFlow;
        this.description = description;
        this.icon = icon;
    }



    public String getWeatherFlow() {
        return weatherFlow;
    }

    public void setWeatherFlow(String weatherFlow) {
        this.weatherFlow = weatherFlow;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getIcon() {
        return icon;
    }

    public void setIcon(int icon) {
        this.icon = icon;
    }

}

File: app/src/main/java/com/baryshev/dmitriy/daresayweather/common/data/weather/pojo/Weather.kt
package com.baryshev.dmitriy.daresayweather.common.data.weather.pojo

import com.google.gson.annotations.SerializedName



data class Weather(
    @SerializedName("id") val id: Int?,
    @SerializedName("main") val main: String?,
    @SerializedName("description") val description: String?,
    @SerializedName("icon") val icon: String?
                  )